pep-0630.rst

```diff
@@ -501,18 +529,20 @@ Module State Access from Slot Methods, Getters and Setters
       you must update ``Py_LIMITED_API`` to ``0x030b0000``, losing ABI
       compatibility with earlier versions.
 
-Slot methods -- the fast C equivalents for special methods, such as
-`nb_add <https://docs.python.org/3/c-api/typeobj.html#c.PyNumberMethods.nb_add>`__
-for ``__add__`` or `tp_new <https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_new>`__
+Slot methods -- the fast C equivalents for special methods, such as `nb_add
+<https://docs.python.org/3/c-api/typeobj.html#c.PyNumberMethods.nb_add>`__
+for ``__add__`` or `tp_new
+<https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_new>`__
 for initialization -- have a very simple API that doesn't allow
-passing in the defining class as in ``PyCMethod``.
+passing in the defining class, as is the case in ``PyCMethod``.
```

Comment:
That sounds verbose. Would “with” work here?
```suggestion
passing in the defining class as with ``PyCMethod``.
```

Reply:
The issue here is that it doesn't come across clearly to me what particular aspect is being implicitly referred to here that "is the case with ``PyCMethod``", and more specifically, whether it is being _compared_ or _contrasted_ with slot methods. Also, the comma is still required to set the phrase off and establish the intended semantic structure. However, an equally short approach that is actually clearer than my original revision is

```suggestion
passing in the defining class, unlike with ``PyCMethod``.
```